### PR TITLE
smtp value settings change for polaris reporting

### PR DESCRIPTION
**Pull request by msenmurugan**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1254#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2020.2.x/OWNERS)~~ [msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1254#issuecomment-589830702):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: pkg/polaris-reporting/ctl_polarisreporting.go [hunk 1]
```diff
@@ -252,36 +252,61 @@ func (ctl *HelmValuesFromCobraFlags) AddHelmValueByCobraFlag(f *pflag.Flag) {
...
 		case "smtp-password":
-			if val, ok := ctl.args["smtp"]; ok && val != nil {
-				smtpVal := val.(map[string]interface{})
-				smtpVal["password"] = ctl.flagTree.SMTPPassword
-				ctl.args["smtp"] = smtpVal
+			if val, ok := ctl.args["onprem-auth-service"]; ok && val != nil {
+				onpremAuthVal := val.(map[string]interface{})
+				if val, ok = onpremAuthVal["smtp"]; ok && val != nil {
+					smtpVal := val.(map[string]interface{})
+					smtpVal["password"] = ctl.flagTree.SMTPUsername
```
Comment (mphammer) [inline pkg/polaris-reporting/ctl_polarisreporting.go]: This should be password right?

Comment (msenmurugan) [reply pkg/polaris-reporting/ctl_polarisreporting.go]: done

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1254#pullrequestreview-362918268" title="LGTM">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1254#" title="Author self-approved">msenmurugan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2020.2.x/OWNERS)~~ [mphammer,msenmurugan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: new pull request created: #1256

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1254#issuecomment-589830702):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**